modules/cluster-samples-operator.adoc

```diff
@@ -1,19 +1,50 @@
 // Module included in the following assemblies:
 //
 // *  operators/operator-reference.adoc
+// *  post_installation_configuration/cluster-capabilities.adoc
 
+// operators/operator-reference.adoc
+ifeval::["{context}" == "cluster-operators-ref"]
+:operator-ref:
+endif::[]
+
+// post_installation_configuration/cluster-capabilities.adoc
+ifeval::["{context}" == "cluster-capabilities"]
+:cluster-caps:
+endif::[]
+
+:_content-type: REFERENCE
 [id="cluster-samples-operator_{context}"]
-= Cluster Samples Operator
+ifdef::operator-ref[= Cluster Samples Operator]
+ifdef::cluster-caps[= OpenShift samples capability]
+
+ifdef::operator-ref[]
+
+[NOTE]
+====
+The Cluster Samples Operator is an optional cluster capability that can be disabled by cluster administrators during installation. For more information about optional cluster capabilities, see "Cluster capabilities" in _Post-installation configuration_.
+====
+
+endif::operator-ref[]
 
 [discrete]
 == Purpose
 
+ifdef::cluster-caps[]
+The Cluster Samples Operator provides the features for the `openshift-samples` capability.
+endif::cluster-caps[]
+
 The Cluster Samples Operator manages the sample image streams and templates stored in the `openshift` namespace.
 
 On initial start up, the Operator creates the default samples configuration resource to initiate the creation of the image streams and templates. The configuration object is a cluster scoped object with the key `cluster` and type `configs.samples`.
 
 The image streams are the {op-system-first}-based {product-title} image streams pointing to images on `registry.redhat.io`. Similarly, the templates are those categorized as {product-title} templates.
 
+ifdef::cluster-caps[]
+If you disable the samples capability, users cannot access the image streams, samples, and templates it provides. Depending on your deployment, you might want to disable this component if you do not need it.
+endif::[]
+
+ifdef::operator-ref[]
```

Comment:
User should be able to get a link with instructions on how to disable the sample capability

Reply:
https://github.com/openshift/openshift-docs/pull/52965 will be resolving this as well. There will be a link to a new page that users encounter prior to installation that details how to disable the capabilities.